tests/e2e/settings.spec.ts

```diff
@@ -72,3 +72,35 @@ test("change code font", async ({ page }) => {
     "jetbrains",
   );
 });
+
+test("change httpd port", async ({ page, peerManager }) => {
```

Comment:
Could we move this test to either `httpd.spec.ts` or `authentication.spec.ts`? It seems like it almost covers https://github.com/radicle-dev/radicle-interface/issues/803 and the port testing is a detail.

Reply:
Yeah, was a leftover from when the port was with the settings modal